### PR TITLE
Add JACK version for Windows

### LINKED ISSUE #739: Link JACK build on Windows page
I think we should link the JACK download on the windows page too (but less prominent without a button) on the Windows install page. 
Any disadvantages?

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**
Adds JACK version to Windows page

**Context: Fixes an issue? Related issues**
Fixes: #739 

**Status of this Pull Request**
Ready for review

**What is missing until this pull request can be merged?**
Needs review

**Does this need translation?**

YES

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [ ] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

File: wiki/en/Installation-for-Windows.md [hunk 1]
```diff
@@ -22,9 +22,13 @@ Upgrading? You may want to [back up your configuration](Software-Manual#backing-
 
 _Please note that you are not permitted to redistribute this binary without acquiring a [licence agreement from Steinberg](https://www.steinberg.net/developers/)._
 
+If you don't want to use ASIO, you can also use the (JACK version described below)(#JACK).
+
 ***
 
-## Jamulus uses [ASIO](https://en.wikipedia.org/wiki/Audio_Stream_Input/Output) 
+## ASIO
```
Comment (ann0see) [inline wiki/en/Installation-for-Windows.md:29]: @gilgongo since we link ASIO in the app, we need to have an anchor with the id ASIO on this page.

File: wiki/en/Installation-for-Windows.md [hunk 2]
```diff
@@ -73,6 +77,15 @@ Have a look at [this video](https://youtu.be/_GzOsitVgLI) by [trombonepizza](htt
...
 
+## JACK
+
+Similar to the Linux version, Jamulus can also use [JACK](https://jackaudio.org/faq/jack_on_windows.html) on Windows. To get started with the JACK version:
+
+1. **Download and install JACK for Windows**. Get it [from the jackaudio page](https://jackaudio.org/downloads/).
+1. [Download Jamulus (JACK version)]({{ site.download_root_link }}{{ site.download_file_names.windows-jack }}){: .button}
+1. Now install Jamulus similar to the Windows ASIO version as mentioned above.
+1. Afterwards, to set up JACK, follow the [Linux setup guide](Installation-for-Linux).
+
```
Comment (ann0see) [inline wiki/en/Installation-for-Windows.md]: We could also move this somewhere else too, but I think it's better to put it at the bottom.

File: wiki/en/Installation-for-Windows.md (hunk 1 above)
Comment (gilgongo) [reply wiki/en/Installation-for-Windows.md:29]: Sorry what do you mean? We have the ASIO rights statement in the site footer now if that's what you mean?

Comment (ann0see) [reply wiki/en/Installation-for-Windows.md:29]: No. The Windows installer links to the headline ASIO via a HTML ID (anchor). So that means we must not name the headline anything other than ASIO.

Comment (gilgongo) [reply wiki/en/Installation-for-Windows.md:29]: Oh OK sure.

File: wiki/en/Installation-for-Windows.md (hunk 2 above)
Comment (gilgongo) [reply wiki/en/Installation-for-Windows.md]: Is there anything we can say about either the JACK or ASIO versions to help someone who might not know which to choose? Could we say ASIO is the preferred version but that JACK may allow more advanced use in some cases? I'm not too sure about the differences myself.

Comment (pljones) [reply wiki/en/Installation-for-Windows.md]: If the user's soundcard ASIO driver is up to scratch and the only ASIO app they use is Jamulus, then there's no reason to use anything but the ASIO version of Jamulus.

If they use other ASIO applications, they have a number of choices: Jack for Windows supports I/O using ASIO, which allows it to be used with other ASIO application without JACK support.  So you could choose to use the ASIO version of Jamulus with JACK anyway.  Or you could use the JACK-native version of Jamulus, which, if you're going to use JACK at all, probably makes more sense.

If the user's soundcard ASIO driver _isn't_ up to scratch, they might find JACK a better option than ASIO4ALL.  There's probably still more help available on the Web regarding ASIO4ALL than JACK for Windows, of course...  So, unless the user is reasonably happy working out how to set up something like ASIO4ALL or JACK for Windows, ASIO4ALL is likely the better choice (their problems will find more help in fixing them).  Where they're happy trying anything, JACK might open up opportunities they'd not thought to try, though.

In simple terms: if you're likely to want to link multiple applications together, JACK _might_ be a good alternative.

Comment (gilgongo) [reply wiki/en/Installation-for-Windows.md]: > if you're likely to want to link multiple applications together, JACK might be a good alternative.

So can we assume 99% of all Windows users have never heard of JACK and don't need to know about it? And the remaining 1% probably know about as part of some specialised thing they're doing, independent of their use of Jamulus. 

In that case, I'd just have a straight download link to the app and cut out the JACK download steps and stuff because that's going to be redundant and draws us into supporting JACK issues. 

Instead, I'd just have a line under the upgrading statement like this:

"If you use [JACK](https://jackaudio.org/) on Windows, we have a version that [supports JACK](link)"

and then the rest of the page is about ASIO. Otherwise it just clogs up the page with stuff that virtually nobody is going to want to know.

Comment (ann0see) [reply wiki/en/Installation-for-Windows.md]: Yes. Probably. However I know some sites proposing that JACK works better than ASIO4ALL due to Port audio support. But I can't verify that JACK works better.

Comment (gilgongo) [reply wiki/en/Installation-for-Windows.md]: Be that as it may, the JACK version is only really relevant if you know about JACK already. Inviting people to "get started" with JACK will get us into trouble if people think its some easy alternative to all the ASIO stuff they need to read though on that page. I'd just put it in as a link and be done with it.

Comment (ann0see) [reply wiki/en/Installation-for-Windows.md]: @gilgongo changed.

**Review by pljones — APPROVED** [review]
